build.gradle

```diff
@@ -77,12 +77,13 @@ project.ext {
 
 apply plugin: "checkstyle"
 checkstyle {
-    toolVersion '8.24'
+    toolVersion '10.3.1'
 }
 
 task checkstyle(type: Checkstyle) {
     classpath = files()
     source "${project.rootDir}"
     exclude("**/gen/**")
     exclude("**/generated/**")
+    exclude("core/src/test/java/android/util/Log.java")
```

Comment:
Shouldn't this already be excluded in the config file here? https://github.com/AntennaPod/AntennaPod/blob/develop/config/checkstyle/suppressions.xml#L6

If it now needs to be excluded in Gradle, it seems like it no longer respects that config file.

Reply:
> it seems like it no longer respects that config file.

Well, according to my tests at least, it never did. I ran this here checkstyle task first without having made any changes, and it returned the same checkstyle violations as it did after the changes. That's why I figured I would add this exclusion anyway, though I should've stated this first.

Reply:
On CI, there are no errors related to that java file. So I think it does indeed use `suppressions.xml` correctly and exclude the file. I would avoid having to specify all exclusions and other rules twice - in gradle and in `suppressions.xml` - and simply only use `suppressions.xml`. Maybe you can try removing the change here and see if it still works on CI?

Reply:
It does work, it's just that the gradle task will return a violation.